openquake/server/templates/engine/includes/footer.html

```diff
@@ -1,11 +1,10 @@
-<footer id="footer" class="footer">                                                                                                                           
+<footer id="footer">        
     <div class="container">
-      <div class="pull-left">
-        <a href="https://github.com/gem/oq-engine" target="_blank">OpenQuake Engine</a> <em>{{ oq_engine_version }}</em> |
-        <a href="{% url "license" %}">License</a> |
-        <a href="https://groups.google.com/forum/#%21forum/openquake-users" target="_blank">Feedback</a>
-      </div>
-      <div class="pull-right">
-      </div> <!-- /.pull-right -->
+        <div class="pull-left">
+            <a href="https://github.com/gem/oq-engine" target="_blank">OpenQuake Engine</a> <em>{{ oq_engine_version }}</em> |
+            <a href="{% url "license" %}">License</a> |
+            <a href="https://groups.google.com/forum/#%21forum/openquake-users" target="_blank">Feedback</a>
+        </div>
+        <div class="pull-right"></div> <!-- /.pull-right -->
```

Comment:
We use 2 spaces indentation in Django templates, not 4, and the whole `<footer>` block must be indented by 2 spaces since it's included (via the base file) in the `<body>` code block, so `<footer>` must be at the indentation level 1 (whereas `<body>` is at 0)

Reply:
Fixed with this commit: https://github.com/gem/oq-engine/pull/5180/commits/a3f6e018e7d920f563c2dbe5b1589fc4a58a9fea